src/DIRAC/Core/Tornado/Client/private/TornadoBaseClient.py

```diff
@@ -63,7 +63,6 @@ class TornadoBaseClient:
     KW_USE_CERTIFICATES = "useCertificates"
     KW_EXTRA_CREDENTIALS = "extraCredentials"
     KW_TIMEOUT = "timeout"
-    KW_SETUP = "setup"
```

Comment:
This cann't be simply removed, it is used later in __discoverSetup() which is called  as initFunc. Those should be also removed

Reply:
Oups, lokked into 8.0, sorry